.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+workspace_root: &workspace_root
+  /go/src/github.com/hashicorp/terraform-aws-consul
+
+defaults: &defaults
+  working_directory: *workspace_root
+  docker:
+  - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:latest
```

Comment:
Just out of interest who owns Account Id `087285199408 `? Do you want to add a comment so future contributors know its a Gruntwork internal account?

Reply:
That's a good idea. @Etiene Could you add a comment explaining this is a Gruntwork Docker image that has `aws`, `jq`, etc installed?